### DISCUSSION TITLE
Piggy Bank - Over-saving

### DISCUSSION THREAD

**Discussion by bartekd123** [firefly-iii/firefly-iii#4356] [Firefly III] [answered]
Hi All,

I am looking to use piggy banks to set aside money for non-monthly expenses in a separate account. I want to continue to save into the piggy banks even if it goes over the goal of the piggy bank. 

Simple example ($60 piggy bank):
 - Bill is 60 every 3 months
 - I set aside 20 every month which is transferred to an account on the first of the month, and linked to the piggy bank
 - on the 3rd month, I have $60 in the account, but the way the paychecks work, it may transfer another $20 into the account by the time I transfer the $60 out of it

Is there a way to work around this and have $80 in the piggy bank, even though $60 is the goal?

**Comment by JC5** [comment, marked as answer]
Only by making the piggy bank bigger, I'm afraid. You can't overfill it.

**Reply by bartekd123** [reply]
Thanks James, I thought that might be the case. Do you think that could be a feature request, so piggy banks can be able to grow past 100%? I will just increase the target amount for each of my piggy banks, but then it throws off all the other numbers.

Or even adding a target amount, as well as a max amount for each of the piggy banks. That would be probably be more work though

**Reply by JC5** [reply]
No, the target amount will remain the maximum amount.

**Reply by bartekd123** [reply]
Thanks James. I’ll work around it. Thanks for your support!
